File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/AnimeCard.java
package com.example.owner_pc.animechecker.model.entity;

import java.util.List;

/**
 * Created by owner-PC on 2017/05/22.
 */

// listviewに表示するアニメ(カードの内容)
public class AnimeCard {
    public AnimePage animePage;
    public Staff director;

    public AnimeCard(AnimePage animePage, Staff director) {
        this.animePage = animePage;
        this.director = director;
    }
    public AnimeCard(AnimePage animePage) {
        this.animePage = animePage;
        this.director = new Staff();
    }
}

File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/AnimeCard.kt
package com.example.owner_pc.animechecker.model.entity

/**
 * Created by owner-PC on 2017/05/22.
 */

// listviewに表示するアニメ(カードの内容)
class AnimeCard {
    var animePage: AnimePage
    var director: Staff

    constructor(animePage: AnimePage, director: Staff) {
        this.animePage = animePage
        this.director = director
    }

    constructor(animePage: AnimePage) {
        this.animePage = animePage
        this.director = Staff()
    }
}
